### DISCUSSION TITLE
Select menu with image dropdown

### DISCUSSION THREAD

**Discussion by sinbadbd** [twbs/bootstrap#32146] [Ideas] [answered]
There is any plans to support select menu with image dropdown in vBS5.  
Like: 
1. https://semantic-ui.com/modules/dropdown.html

**Comment by patrickhlauke** [comment, marked as answer]
no plans at this stage, particularly because to do them properly (i.e. both keyboard accessible and correctly conveyed to assistive technologies) is non-trivial (that semantic-ui version, for instance, fails miserably in screen readers)

**Reply by sinbadbd** [reply]
Okay got it, Thanks!
